nabs/utils.py

```diff
@@ -110,3 +116,108 @@ def add_named_kwargs_to_signature(
     )
 
     return sig.replace(parameters=start_params + wrapper_params + end_params)
+
+
+def format_ophyds_to_html(obj, allow_child=False):
+    """
+    Recursively construct html that contains the output from .status() for
+    each object provided.  Base case is being passed a single ophyd object
+    with a `.status()` method.  Any object without a `.status()` method is
+    ignored.
+
+    Creates divs and buttons based on styling
+    from `nabs._html.collapse_list_head` and `nabs._html.collapse_list_tail`
+
+    Parameters
+    ----------
+    obj : ophyd object or Iterable of ophyd objects
+        Objects to format into html
+
+    Returns
+    -------
+    out : string
+        html body containing ophyd object representations (sans styling, JS)
+    """
+    if isinstance(obj, Iterable):
+        content = ""
+
+        for o in obj:
+            content += format_ophyds_to_html(o, allow_child=allow_child)
+
+        # Don't return wrapping if there's no content
+        if content == "":
+            return content
+
+        # HelpfulNamespaces tend to lack names, maybe they won't some day
+        parent_name = getattr(obj, '__name__', 'expand me')
```

Comment:
Maybe something like `"Ophyd status: " + " ".join(o.name for o in obj)` (and truncate beyond 60 chars or something) for the 'expand me' default?